libmultilabel/nn/nn_utils.py

```diff
@@ -170,7 +171,7 @@ def set_seed(seed):
 
     if seed is not None:
         if seed >= 0:
-            seed_everything(seed=seed)
+            seed_everything(seed=seed, workers=True)
             torch.use_deterministic_algorithms(True)
             torch.backends.cudnn.benchmark = False
```

Comment:
Check if we still need these two lines.

Reply:
These lines are not necessary. Updated, thanks!